Shop/ViewModel/CarListViewModel.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Shop.Model;
+using Shop.Model.Constant;
+using Shop.ViewModel.Commands;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace Shop.ViewModel
+{
+    public class CarListViewModel : INotifyPropertyChanged
```

Comment:
ViewModel name should follow control/window name the VM belongs too.
In that case - MainWindowViewModel.

Reply:
-1 to source code quality.